### PR TITLE
Syntax extension to allow specifying chaining in call_m

### PR DESCRIPTION
**Pull request by ddeclerck**

### CODE REVIEW

**Review by Keryan-dev — CHANGES_REQUESTED** [review]
I'd like to see the changes in mpp_specs/ also. 
Otherwise it's all good to me.

**Comment by ddeclerck** [comment]
> I'd like to see the changes in mpp_specs/ also. Otherwise it's all good to me.

It should be alright now.

**Review by Keryan-dev — APPROVED** [review]

File: src/mlang/mpp_ir/mpp_ir_to_bir.ml [hunk 1]
```diff
@@ -342,7 +342,7 @@ and translate_mpp_stmt (mpp_program : Mpp_ir.mpp_compute list)
       translate_mpp_function mpp_program m_program
         (List.find (fun decl -> decl.Mpp_ir.name = f) mpp_program)
         real_args ctx
-  | Mpp_ir.Expr (Call (Program, args), _) ->
+  | Mpp_ir.Expr (Call (Program _chain, args), _) ->
```
Comment (denismerigoux) [inline src/mlang/mpp_ir/mpp_ir_to_bir.ml:345]: Ok this means that for now we don't do anything with the chain information and we just call all the rules ?

**Review by denismerigoux — APPROVED** [review]

File: src/mlang/mpp_ir/mpp_ir_to_bir.ml (hunk 1 above)
Comment (ddeclerck) [reply src/mlang/mpp_ir/mpp_ir_to_bir.ml:345]: Yup. Keryan needs this in his branch.

**Pull request merged**